Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPCorefinementVisitor.h

```diff
@@ -33,28 +33,50 @@ typedef unspecified_type halfedge_descriptor;
   void after_subface_created(face_descriptor f_new, Triangle_mesh& tm);
 /// @}
 
-/// @name Functions used by corefine() when edges are split
+/// @name Functions used by corefine() when edges are split or created
```

Comment:
backtick `corefine()` ??  Or ugly in a heading?

Reply:
Indeed, I think backticks are not supported in heading